CourseManager/Courses/Course 1 Build a SwiftUI app for iOS 15/Model/Coin.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  Coin.swift
+//  CourseManager
+//
+//  Created by Summer Crow on 2022-11-11.
+//
+
+import Foundation
+
+struct Coin: Identifiable, Decodable {
+    var id: Int
+    var coin_name: String
```

Comment:
Use always camelCase

Reply:
This is just embarrassing. I was blindly following the instructor, would never do this in my own app. I've updated to include CodingKeys